OloEngine/src/OloEngine/Audio/AudioListener.cpp

```
@@ -7,34 +7,34 @@
+            ::ma_engine_listener_set_world_up(engine, m_ListenerIndex, 0, 1, 0);
+            setupWorldUp = true;
+        }
+    }
 
-	void AudioListener::SetVelocity(const glm::vec3& velocity) const
-	{
-		auto* engine = (ma_engine*)AudioEngine::GetEngine();
-		::ma_engine_listener_set_velocity(engine, m_ListenerIndex, velocity.x, velocity.y, velocity.z);
-	}
-}
+    void AudioListener::SetDirection(const glm::vec3& forward) const
```

Clang (14.0.6)

[warning] method 'SetDirection' can be made static

(readability-convert-member-functions-to-static)